pom.xml

```diff
@@ -7,7 +7,7 @@
 
 	<parent>
 		<groupId>net.sansa-stack</groupId>
-		<artifactId>sansa-parent</artifactId>
+		<artifactId>sansa-parent_2.11</artifactId>
```

Comment:
basically, the PR itself prepared for releasing different Scala version used as suffix of the artifact name. So yes, as we move to Scala 2.12, it will then be changed to `sansa-parent_2.12` which contains the Scala version properties for 2.12